From: Mitsutoshi NAKANO <ItSANgo@gmail.com>
Date: Sun, 21 Jun 2015 19:41:39 +0900
Subject: fix-print_msg-macros

* Wnn/uum/sdefine.h: added Copyright 2015.
(print_msg): fixed printf(X) -> fputs((X), stdout).
(print_msg_getc): fixed printf(X) -> fputs((X), stdout).

* ChangeLog.en: added entries.
* ChangeLog: added entries.
---
 ChangeLog         | 9 +++++++++
 ChangeLog.en      | 9 +++++++++
 Wnn/uum/sdefine.h | 6 +++---
 3 files changed, 21 insertions(+), 3 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index 1004a70..fdf5ade 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -3,6 +3,15 @@
   Japanese...
  */
 
+Sun, 21 Jun 2015 19:23:13  Mitsutoshi NAKANO  <ItSANgo@gmail.com>
+
+	* Wnn/uum/sdefine.h: added Copyright 2015.
+	(print_msg): fixed printf(X) -> fputs((X), stdout).
+	(print_msg_getc): fixed printf(X) -> fputs((X), stdout).
+
+	* ChangeLog.en: added these entries.
+	* ChangeLog: added these entries.
+
 Sun, 21 Jun 2015 11:59:19  Mitsutoshi NAKANO  <ItSANgo@gmail.com>
 
 	* Wnn/uum/select_ele.c: added Copyright 2015.
diff --git a/ChangeLog.en b/ChangeLog.en
index 2566f7d..d3623ee 100644
--- a/ChangeLog.en
+++ b/ChangeLog.en
@@ -1,3 +1,12 @@
+Sun, 21 Jun 2015 19:23:13  Mitsutoshi NAKANO  <ItSANgo@gmail.com>
+
+	* Wnn/uum/sdefine.h: added Copyright 2015.
+	(print_msg): fixed printf(X) -> fputs((X), stdout).
+	(print_msg_getc): fixed printf(X) -> fputs((X), stdout).
+
+	* ChangeLog.en: added these entries.
+	* ChangeLog: added these entries.
+
 Sun, 21 Jun 2015 11:59:19  Mitsutoshi NAKANO  <ItSANgo@gmail.com>
 
 	* Wnn/uum/select_ele.c: added Copyright 2015.
diff --git a/Wnn/uum/sdefine.h b/Wnn/uum/sdefine.h
index 305c4cc..70a805d 100644
--- a/Wnn/uum/sdefine.h
+++ b/Wnn/uum/sdefine.h
@@ -10,7 +10,7 @@
  *                 1987, 1988, 1989, 1990, 1991, 1992
  * Copyright OMRON Corporation. 1987, 1988, 1989, 1990, 1991, 1992, 1999
  * Copyright ASTEC, Inc. 1987, 1988, 1989, 1990, 1991, 1992
- * Copyright FreeWnn Project 1999, 2000, 2006
+ * Copyright FreeWnn Project 1999, 2000, 2006, 2015
  *
  * Maintainer:  FreeWnn Project
  *
@@ -191,8 +191,8 @@ struct kansuu
 
 
 #define flush() fflush(stdout)
-#define print_msg(X) {push_cursor();throw_c(0); clr_line();printf(X);flush();pop_cursor();}
-#define print_msg_getc(X) {push_cursor();throw_c(0); clr_line();printf(X);flush();keyin();pop_cursor();}
+#define print_msg(X) {push_cursor();throw_c(0); clr_line();fputs((X), stdout);flush();pop_cursor();}
+#define print_msg_getc(X) {push_cursor();throw_c(0); clr_line();fputs((X), stdout);flush();keyin();pop_cursor();}
 
 
 #define UNDER_LINE_MODE (0x02 | 0x08 | 0x20)
